components/automate-chef-io/content/docs/backup.md

```diff
@@ -188,69 +188,81 @@ where `bucket_name` is the name of the S3 bucket `base_path` an optional path wi
 
 Restore Chef Automate to an instance on which Automate is not already installed.
 
-### Restore From a Filesystem Backup
-
-#### Prerequisites
+### Prerequisites
 
 1. On the restore host, download and unzip the Chef Automate command-line tool:
-    ```shell
+
+   ```shell
         curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
     ```
 
-1. Ensure access for Chef Automate to a backup directory in the [configured location]({{< ref
-"backup.md#backup-to-filesystem" >}}):
+1. *(filesystem backups only)* Ensure access for Chef Automate to a backup directory in the [configured location]({{< ref
+"backup.md#backup-to-a-filesystem" >}}):
 
      1. To restore a network-attached filesystem backup, mount the shared backup directory to the same mount point configured at the time of the backup.
      1. To restore a backup directory that is not a network-attached filesystem, copy the backup directory to the location that was [configured]({{< ref
-"backup.md#backup-to-filesystem" >}}) at the time of the backup.
-     1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-filesystem" >}}).
+"backup.md#backup-to-a-filesystem" >}}) at the time of the backup.
+     1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-a-filesystem" >}}).
 
-#### Restore in an Internet-Connected Environment
+1. *(different FQDN only)* To restore to a host whose FQDN differs from that of the host on which the backup was
```

Comment:
I don't love the italics here; happy to go with whatever convention we use to emphasize text.